src/backends/sentry_backend_crashpad.cpp

```diff
@@ -123,8 +123,8 @@ sentry__crashpad_backend_startup(sentry_backend_t *backend)
     sentry__crashpad_backend_user_consent_changed(backend);
 
     crashpad::CrashpadClient client;
-    char *minidump_url = sentry__dsn_get_minidump_url(&options->dsn);
-    std::string url(minidump_url);
+    auto minidump_url = sentry__dsn_get_minidump_url(&options->dsn);
+    auto url = minidump_url ? std::string(minidump_url) : std::string();
```

Comment:
👍 thats a good one! I actually stumbled on this crash recently but didn’t fix it myself yet.
I would have to consult with @jan-auer about how to deal with this. We were discussing to have crashpad dump stuff to disk instead of uploading itself. Because without any url you can hand it, its pretty much useless, right?

Reply:
I did not really think about the intrinsic logic behind this tbh, but I agree we should maybe test the pointer and throw some exception instead.